_source/_posts/2020-01-21-angular-material-login.md

```diff
@@ -40,67 +41,56 @@ In this tutorial, I'll show you how to create a login form like the one above. Y
+Depending on your operating system, you might have to run this using the `sudo` command. This will install the Angular command-line tool on your system. It lets you use the `ng` command to set up and manipulate Angular applications. In order to create an Angular app for the version we want, we'll use `npx` to create a new app. To create a new application navigate to a directory of your choice and run the following.
 
 ```bash
-ng new material-tic-tac-toe
+npx @angular/cli@13 new material-tic-tac-toe
 ```
 
-You will be asked three questions. Answer **Yes** to the first two questions. This will enable stricter type checking and set up the router in your application. The router is responsible for letting the user navigate between different parts of the app and updating the browser's URL without actually reloading the page. For the third question, accept the default choice **CSS**. This application will use simple CSS stylesheets, but you can see how easy it is to switch to a different stylesheet technology with Angular.
+You will be asked two questions. Answer **Yes** to the first question. This will set up the router in your application. The router is responsible for letting the user navigate between different parts of the app and updating the browser's URL without actually reloading the page. For the third question, accept the default choice **CSS**. This application will use simple CSS stylesheets, but you can see how easy it is to switch to a different stylesheet technology with Angular.
 
 {% img blog/angular-material-login/angular-style-options.png alt:"Angular stylesheet options" width:"700" %}{: .center-image }
 
 Once the wizard has completed you should see a new folder called `material-tic-tac-toe`. Navigate into the folder and install some packages you will need for the application by running the command below.
 
 ```bash
-npm install -E @angular/material@11.2.6 @angular/flex-layout@11.0.0-beta.33 hammerjs@2.0.8 \
-  @angular/cdk@11.2.6 tic-tac-toe-minimax@1.0.8
+npm install -E @angular/flex-layout@13.0.0-beta.36 tic-tac-toe-minimax@1.0.8
 ```
 
-The `@angular/material` package provides the components of the Material Design, `@angular/cdk` is a component development kit that is needed for the Material components to work and `hammerjs` is a library that provides smooth animations for the component. `@angular/flex-layout` provides a flexible and responsive grid. It is independent of the Material components but is often used together with it. Finally, `tic-tac-toe-minimax` is a ready-to-go tic-tac-toe game with a computer player.
+`@angular/flex-layout` provides a CSS layout system supporting both FlexBox and CSS Grid using Angular directives. It is independent of the Material components but is often used together with it. And, `tic-tac-toe-minimax` is a ready-to-go tic-tac-toe game with a computer player.
 
-## Add Angular Material CSS
```

Comment:
If remove the keywords (Angular Material) in a header, maybe they should be added to another header?

Reply:
@mraible The words "Angular Material" are in 5 headers, plus the title. I can find a way to incorporate it if we need more, but I want to double-check you think this is needed since it's in quite a few headers already.

Reply:
That's probably enough. I typically aim for 2-3 headers with the keywords.